src/applications/vaos/components/FutureAppointmentsList.jsx

```diff
@@ -150,7 +150,7 @@ export class FutureAppointmentsList extends React.Component {
     }
 
     const header = (
-      <h3 className="vads-u-margin-y--4">Upcoming appointments</h3>
```

Comment:
All the list items use h3 headers, and the main title uses h1, not sure why this was changed to h3 at some point.